pkg/enqueue/Symfony/DefaultTransportFactory.php

```diff
@@ -161,7 +162,11 @@ private function resolveDSN(ContainerBuilder $container, $dsn)
             $matches = [];
             if (preg_match('/%env\((.*?)\)/', $dsn, $matches)) {
                 if (false === $realDsn = getenv($matches[1])) {
-                    throw new \LogicException(sprintf('The env "%s" var is not defined', $matches[1]));
+                    try {
+                        $realDsn = $container->getParameter($matches[1]);
```

Comment:
shouldn't it be `$dsn` instead of `$matches[1]`

Reply:
@Steveb-p ping ^^

Reply:
I'll take a look at it tonight :)